lib/dockerregistry/inventory.go

```diff
@@ -475,7 +475,7 @@ func (sc *SyncContext) IgnoreFromPromotion(regName RegistryName) {
 		klog.Errorf("unable to ignore from promotion: %s\n", err)
 	}
 
-	klog.Errorf("ignoring from promotion: %s\n", err)
+	klog.Infof("ignoring from promotion: %s\n", imgName)
```

Comment:
Do you want to log the error here?  (If the answer is "the errors are going to be obvious by looking at the registry name", that is a fine answer and I agree 100% there is no point logging the error)

Reply:
Actually the error is already logged (in line 475) with Errorf; I just wanted to add an another informational message, which is why I used Infof instead of Errorf (although maybe I should have used Warningf instead).